homeassistant/components/eight_sleep/__init__.py

```diff
@@ -168,9 +189,27 @@ def __init__(
 
         mapped_name = NAME_MAP.get(sensor, sensor.replace("_", " ").title())
```

Comment:
There isn't anything wrong with this, but stylewise I'd write it to avoid the multi-line statements
```python3
        mapped_name = NAME_MAP.get(sensor, sensor.replace("_", " ").title())
        if self._user_obj is not None:
            name = f"{self._user_obj.user_profile['firstName']}'s {mapped_name}"
            self._attr_name = name
        else:
            self._attr_name = f"Eight Sleep {mapped_name}"
        unique_id = f"{_get_device_unique_id(eight, self._user_obj)}.{sensor}"
        self._attr_unique_id = unique_id
```